tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/02_productBlock.js

```diff
@@ -587,7 +596,7 @@ describe('BO - Orders - View and edit order : Check product block in view order
       it('should check the ordered product details', async function () {
         await testContext.addContextItem(this, 'testIdentifier', 'checkProductOutOfStockAllowedDetails', baseContext);
 
-        const result = await orderPageProductsBlock.getProductDetails(page, 1);
+        const result = await orderPageProductsBlock.getProductDetails(page, 4);
```

Comment:
```suggestion
        const result = await orderPageProductsBlock.getProductDetails(page, 5);
```

Shoudn't it be 5 logically, since the previous modifications kept increasing except 5 which is missing

Reply:
Hi @jolelievre,

This product will be on line 4 because the reference of the last product added in line 4 is **demo_4**, and this one has the reference **d12345** => it will take line 4 and **demo_4** will take line 5 :)